src/parser/mod.rs

```diff
@@ -6083,6 +6083,21 @@ impl<'a> Parser<'a> {
                 None
             };
             Ok(Some(ColumnOption::Identity(property)))
+        } else if dialect_of!(self is SQLiteDialect)
```

Comment:
```suggestion
        } else if dialect_of!(self is SQLiteDialect | GenericDialect)
```
we can extend the generic dialect if it doesn't conflict with existing behavior?

Reply:
Extending the GenericDialect doesn't conflict with existing behaviour.

But for example this syntax doesn't work with a Postgres database (and I suspect many others).

But if you think it is proper to extend the GenericDialect (I have no opinion on this matter) I shall do it.